src/mlpack/methods/ann/layer/gru.hpp

```diff
@@ -72,6 +72,19 @@ class GRU
       const size_t outSize,
       const size_t rho = std::numeric_limits<size_t>::max());
 
+  // Copy constructor.
+  GRU(const GRU& layer);
+
+  // Move constructor.
+  GRU(GRU&&);
+
+  // Copy assignment operator.
+  GRU &operator=(const GRU& layer);
```

Comment:
```suggestion
  GRU& operator=(const GRU& layer);
```
To match the style in other files.